packages/cli/src/commands/auth/login.ts

```diff
@@ -30,18 +30,18 @@ export default class Login extends Command {
       ],
       default: 'us-east-1',
       description:
-        'The platform region to log into. See https://docs.coveo.com/en/2976.',
+        'The Coveo Platform region to log in to. See <https://docs.coveo.com/en/2976>.',
```

Comment:
🤔 what's the point of the angle brackets?

Reply:
If I want to output these as hyperlinks on the docs site, our Markdown parser/converter will recognize `<https://example.com>` syntax as a link that doesn't have a label and convert it to `<a href="https://example.com"></a>`

Otherwise, if I just leave the link as is, it's going to be displayed as plain text on the docs site.